### ISSUE TITLE
Pylint don't work with `coc-pyright` v1.1.144

### ISSUE THREAD

**Issue by lioaslan** [fannheyward/coc-pyright#505]
**What's the output of `:CocCommand pyright.version`**: 1.1.144
`pylint`: 2.8.2

Yesterday, My Linux had some problems so I reinstalled Linux, including `coc-pyright` and then `pylint` didn't work with it. It doesn't show any error or warning. 
Additionally, when I removed all coc settings about python, it worked with `pyright`

Following #480,  here is my `coc.setting`
```json
{
  "clangd.enabled": true,
  "clangd.path": "/home/leo/.config/coc/extensions/coc-clangd-data/install/11.0.0/clangd_11.0.0/bin/clangd",
  "diagnostic.displayByAle": true,
  "python.linting.enabled": true,
  "python.linting.pylintEnabled": true,
  "pyright.disableDiagnostics": true
}
```
Thanks for any help!!

**Comment by and3rson** [comment]
Did you figure this out? I still cannot get pylint to work with coc-pyright.

**Comment by fannheyward** [comment]
@and3rson open a new issue with your `:CocCommand workspace.showOutput coc-pyright-linting` and `:CocCommand workspace.showOutput Pyright`

**Comment by lioaslan** [comment]
@and3rson Yes, I did. I forgot how I solved it but that's my fault

**Comment by and3rson** [comment]
I've just figured it out. It looks like I needed to install pylint inside venv. coc-pyright was not using the global one.

Thanks for the help anyway!
